sample_metadata/parser/generic_metadata_parser.py

```diff
@@ -487,6 +486,9 @@ async def get_sequence_meta(
         for r in rows:
             if self.reads_column and self.reads_column in r:
                 read_filenames.extend(r[self.reads_column].split(','))
+            if self.reads_column is None:
+                filenames = await self.get_read_filenames(sample_id=sample_id, row=r)
+                read_filenames.extend(filenames)
```

Comment:
Can this just be called directly instead - this logic is better placed in that method anyway, then cleaner to override.

```suggestion
            reads = self.get_read_filenames(sample_id=sample_id, row=r)
            if reads:
                read_filenames.extend(read_filenames)
```